quickannotator/api/v1/image/routes.py

```diff
@@ -50,29 +55,187 @@ def delete(self, args):
         return 204
 
 #################################################################################
-@bp.route('/<int:project_id>/search', endpoint="image_search")
-class ImageSearch(MethodView):
-    @bp.arguments(server_models.SearchImageArgsSchema, location='query')
-    @bp.response(200, server_models.ImageRespSchema(many=True))
-    def get(self, args, project_id):
-        """     returns a list of Images
-        """
-        result = db_session.query(db_models.Image).filter(db_models.Image.project_id == project_id).all()
-        return result, 200
-
+    @bp.route('/<int:project_id>/search', endpoint="image_search")
+    class ImageSearch(MethodView):
+        @bp.arguments(server_models.SearchImageArgsSchema, location='query')
+        @bp.response(200, server_models.ImageRespSchema(many=True))
+        def get(self, args, project_id):
+            """     returns a list of Images
+            """
+            result = db_session.query(db_models.Image).filter(db_models.Image.project_id == project_id).all()
+            return result, 200
+    @bp.route('/project/<int:project_id>', endpoint="images")
+    class ImageSearch(MethodView):
```

Comment:
why is this a "search" and not just a "get"?

Reply:
Change the name to "getImagesByProjectId".